perfkitbenchmarker/scripts/object_storage_api_tests.py

```diff
@@ -354,7 +384,8 @@ def WriteObjects(storage_schema, bucket, object_prefix, count,
     host_to_connect: An optional endpoint string to connect to.
   """
 
-  payload_bytes, payload_string = GenerateWritePayload(size)
+  payload = GenerateWritePayload(size)
+  handle = cStringIO.StringIO(payload)
```

Comment:
have you confirmed whether creation of the handle creates a copy of the payload?